docs/modules/user-guide/partials/con_using-a-top-level-attribute.adoc

```diff
@@ -0,0 +1,38 @@
+[id="con_using-top-level-attribute_{context}"]
```

Comment:
We are we saying `using` when we're adding a top level attribute? 

Also, this doc seems more like a procedure than a concept module. The point is to instruct users to add a top level attribute which is procedure, not concept. 

docs/modules/user-guide/partials/proc_adding-a-top-level-attribute.adoc
[id="proc_adding-a-top-level-attribute_{context}"]
= Adding a top-level attribute